HHS/Head-Start-TTADP
# [TTAHUB-690/743] Goals and objectives (on Activity Report) page redesign

## Description of change

Changing the goals & objectives tab on the AR to use the new fields. This change includes the new frontend for both recipient and other entity reports.

This is for the **frontend** only! Every time you save, your report will be in a nuked state so please bear that in mind while testing.

A future PR will include the radio buttons/file uploader.

## How to test

On the goals & objectives tab, you can add and edit goals & objectives. The validation works in a similar way to the existing AR form. You will not be able to save any data and return to it. If your report autosaves, and you navigate away from a tab, you will find your goals & objectives all gunked up.

Additionally, form controls conform to the logic here: https://docs.google.com/document/d/1AG1Zycwgapeeo2gjVkW4uAwF9x7VRZhHqFiwJQCK2as/edit - note that the backend doesn't allow right now for the behavior we want in topics and resources, so that's an exception

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-690
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-743

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
